technic/tools/mining_drill.lua

```diff
@@ -246,20 +246,28 @@ local function pos_is_pointable(pos)
 	return nodedef and nodedef.pointable
 end
 
+local function replaceMk(str, mk, mode)
+	local t = {}
+	t['1'] = mk
+	t['2'] = mode
+	local str1 = str
+	local str2 = string.gsub(str1, "%%(%d+)", t)
+	return str2
+end
```

Comment:
Shorter, and a more intuitive function name.
```suggestion
local function get_description_string(str, mk, mode)
	local t = { ["1"] = mk, ["2"] = mode }
	return (string.gsub(str, "%%(%d+)", t))
end
```